high_card.rb

```diff
@@ -1,5 +1,59 @@
 # Use these two arrays to generate a deck of cards.
-ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
-suits = [ "hearts", "spades", "clubs", "diamonds" ]
+class Game
+  attr_accessor :deck, :shuffled_deck, :players, :hand
+  def initialize
+    @deck = []
+    @shuffled_deck = []
+    @players = []
+    @hand = []
+  end
+  def build_deck
+    ranks = [2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K", "A"]
+    suits = [ "hearts", "spades", "clubs", "diamonds" ]
 
-players = []
+    ranks.each_with_index do |rank, i|
+      suits.each do |suit|
+        @deck << ([rank , suit, i])
```

Comment:
this works, but I think pushing an object is a little more optimal for future parsing of values